exercises/secret-handshake/src/example/kotlin/Signal.kt

```diff
@@ -0,0 +1,5 @@
+enum class Signal {
```

Comment:
Looks like this caused the unit test compilation to fail. Maybe I unwittingly changed something while consolidating with the Java track. Looking...